### PR TITLE
Feature/offline maps

### PR DESCRIPTION
**Pull request by bholota**
### Does any other open PR do the same thing?
Probably no. I was requested to assist with offline maps to our in house RN team (I'm native dev).

### What issue is this PR fixing?
Library user may decide to build map with tiles stored manually in internal storage. It's all based on features already present in OSM native implementation.

### How did you test this PR?
It was used in app developed for our client, so far it works. You may test it LocalCustomTiles.js that may be started from sample list. But first you have to place proper zip with tiles here: `/data/data/{packageName}/files/offline_tiles/`

### CODE REVIEW

File: lib/components/MapFileTile.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  }
+}
+
+MapFileTile.propTypes = propTypes;
+
+export default decorateMapComponent(MapFileTile, {
+  componentType: 'FileTile',
+  providers: {
+    google: {
+      ios: SUPPORTED,
```
Comment (fqborges) [inline lib/components/MapFileTile.js]: This lines (72-73) means that offline tiles are compatible with google maps on android and ios. Unless I am missing something, it seems to not be the case.

**Review by fqborges — COMMENTED** [review]
Awesome job. I have just one comment.

File: lib/components/MapFileTile.js (hunk 1 above)
Comment (bholota) [reply lib/components/MapFileTile.js]: You're right about this. I just based it on MapUrlTile.js, without going into details. Google map provider shouldn't be supported at all. I don't know react and js part in this lib well - should I remove entire `google` block?

Comment (fqborges) [reply lib/components/MapFileTile.js]: Yes, please. Thanks.

Comment (bholota) [reply lib/components/MapFileTile.js]: Done.

**Pull request merged**